libs/ui/src/utils/redirecit-to-maximise-view.util.ts

```diff
@@ -1,9 +1,13 @@
+import { OnEventFn } from '@rnw-community/shared';
+
 import { isFullpage } from './location.utils';
-import { openMaximiseScreen } from './open-maximise-screen.util';
+import { openFullViewPage } from './open-maximise-screen.util';
 import { isWeb } from './platform.utils';
 
-export const redirectToMamixiseView = () => {
+export const redirectToFullViewPage = (fn: OnEventFn<void>) => {
+  fn();
+
   if (isWeb && !isFullpage) {
-    openMaximiseScreen();
+    setTimeout(openFullViewPage, 0);
```

Comment:
you could skip 0 if you don't need to specify delay, leave just function inside setTimeout